src/Symfony/Component/Workflow/Dumper/GraphvizDumper.php

```diff
@@ -120,8 +120,8 @@ protected function addTransitions(array $transitions)
     {
         $code = '';
 
-        foreach ($transitions as $place) {
-            $code .= sprintf("  transition_%s [label=\"%s\", shape=box%s];\n", $this->dotize($place['name']), $place['name'], $this->addAttributes($place['attributes']));
+        foreach ($transitions as $i => $place) {
+            $code .= sprintf("  transition_%d [label=\"%s\", shape=box%s];\n", $this->dotize($i), $place['name'], $this->addAttributes($place['attributes']));
```

Comment:
@lyrixx We could remove the call to `dotize` here as `$i` is always an integer, right?

Reply:
Nevermind, in 4.2, we are using a hash, so a string (so I'm going to change `%d` to `%s` in 4.2).